### PR TITLE
DOC: small doc and formulation changes

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
Including CSS, to put the download note of the examples on the side.

### CODE REVIEW

**Comment by GaelVaroquaux** [comment]
Ping @illdopejake for a review.

File: examples/03_connectivity/plot_compare_decomposition.py [hunk 1]
```diff
@@ -120,28 +115,21 @@
...
-for estimator, components in zip(estimators, components_imgs):
-    # 4D plotting
-    plot_prob_atlas(components, view_type="filled_contours",
-                    title="%s" % names[estimator],
-                    cut_coords=cut_coords, colorbar=False)
-    # 3D plotting
-    plot_stat_map(index_img(components, indices[estimator]),
-                  title="%s" % names[estimator],
-                  cut_coords=cut_coords, colorbar=False)
+#
```
Comment (illdopejake) [inline examples/03_connectivity/plot_compare_decomposition.py]: If we remove this part, it's going to break the resting_state_networks.rst in the User Guide. It's fine if you want to remove it but we'll also need to edit that doc as well. (As of now, it's linking to the images created in the lines you removed).

**Comment by GaelVaroquaux** [comment]
> If we remove this part, it's going to break the resting_state_networks.rst in
> the User Guide. It's fine if you want to remove it but we'll also need to edit
> that doc as well. (As of now, it's linking to the images created in the lines
> you removed).

Good point. I need to fix that.

**Comment by GaelVaroquaux** [comment]
I addressed your comment @illdopejake, and reworked a bit that page. Let's wait for CI.

File: doc/connectivity/resting_state_networks.rst [hunk 2]
```diff
@@ -84,6 +90,14 @@ We can also plot the map for different components separately:
    a fair representation, you should display all components and
    investigate which one resemble those displayed above.
 
+Interpreting such components
+-----------------------------
+
+ICA, and related algorithms, extract pattern that coactivate in the
```
Comment (illdopejake) [inline doc/connectivity/resting_state_networks.rst]:
```suggestion
ICA, and related algorithms, extract patterns that coactivate in the
```

File: doc/connectivity/resting_state_networks.rst [hunk 3]
```diff
@@ -84,6 +90,14 @@ We can also plot the map for different components separately:
...
+signal. As a result, it find functional networks, but also patterns of
```
Comment (illdopejake) [inline doc/connectivity/resting_state_networks.rst]:
```suggestion
signal. As a result, it finds functional networks, but also patterns of
```

**Comment by illdopejake** [comment]
@GaelVaroquaux I couldn't identify just from looking at it what might've broken circle CI. I'm going to try to build it locally to see if I can figure it out

**Comment by illdopejake** [comment]
Scratch that -- looks like it's running?

**Comment by GaelVaroquaux** [comment]
Yes, I just stopped the first build, as it was redundant. I think that this is a merge, if you're ok with it.

**Comment by illdopejake** [comment]
Yep let's do it :-)

**Pull request merged**